### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by jesustorresdev** [stnrd/healthicons#15]
Hi,

I haven't been able to find the components for the next icons:

~~~
people/diabetes
people/speech_language_therapy
specialties/pharmacy
conditions/sweating
devices/fever
devices/eyeglasses
~~~

I guess it is because there are other icons with the same component name, so the components for those icons overwrite the components of the previous ones.

For example, `devices/fever` and `emotion/fever` use the same component name. Another example es `people/diabetes` and `device/diabetes`

**Comment by stnrd** [comment]
Aah that is probably the case I will have a look tomorrow to fix this. I think it's best to suffix the icon category for these duplicates. I prefer to do it only for these duplicates and not pollute the other icon names. What do you think ?

I will pick up the typo as well.

**Comment by jesustorresdev** [comment]
Yes, I agree. It is better only do that where there are a name conflict.

**Comment by sten-nico** [comment]
Solved in #16
